<ui version="4.0" >
 <class>qdFileOperation</class>
 <widget class="QDialog" name="qdFileOperation" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>267</width>
    <height>145</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QLabel" name="qlCount" >
     <property name="text" >
      <string>TextLabel</string>
     </property>
     <property name="buddy" >
      <cstring>qcbDestination</cstring>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QComboBox" name="qcbDestination" >
       <property name="editable" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="qpbBrowse" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Maximum" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>&amp;Browse</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>&amp;Filter:</string>
     </property>
     <property name="buddy" >
      <cstring>qcbFilter</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="qcbFilter" >
     <property name="editable" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="qpbOK" >
       <property name="text" >
        <string>&amp;OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="qpbEnqueue" >
       <property name="text" >
        <string>&amp;Enqueue</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="qpbCancel" >
       <property name="text" >
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>qpbOK</tabstop>
  <tabstop>qpbEnqueue</tabstop>
  <tabstop>qpbCancel</tabstop>
  <tabstop>qpbBrowse</tabstop>
  <tabstop>qcbDestination</tabstop>
  <tabstop>qcbFilter</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
